### ISSUE TITLE
build failed for Darwin/arm64

### ISSUE THREAD

**Issue by emar-kar** [thatisuday/go-cross-build#5]
Build failed without any explanation.

2021-04-18T20:22:55.9499410Z An error occurred during build: exit status 2
2021-04-18T20:22:55.9504744Z exit status 1

**Comment by emar-kar** [comment]
I didn't encounter this error on my local machine:
![image](https://user-images.githubusercontent.com/46078689/115464095-3751b480-a235-11eb-876d-6cbc5282d80e.png)

From *Go 1.16* release notes.
```
Go 1.16 adds support of 64-bit ARM architecture on macOS (also known as Apple Silicon) with GOOS=darwin, 
GOARCH=arm64. Like the darwin/amd64 port, the darwin/arm64 port supports cgo, internal and external linking, 
c-archive, c-shared, and pie build modes, and the race detector.

The iOS port, which was previously darwin/arm64, has been renamed to ios/arm64. GOOS=ios implies 
the darwin build tag, just as GOOS=android implies the linux build tag. This change should be transparent 
to anyone using gomobile to build iOS apps.
```

Nevertheless, those flags should build binary for iOS platform without any error. I would like to test action with the local build on linux system and try to get the exact error. But, from what I can gather it can depends on *Go* version.

### ADDITION
[Failed build](https://github.com/emar-kar/course-project/runs/2394571655?check_suite_focus=true)

**Comment by williampiv** [comment]
Tested with a newer version of Golang (and Debian), and can confirm that does fix this issue. Was PR #9 , closed since #8 is already there.
